magento2-nginx/fpm-7.0/usr/local/share/magento2/install_magento.sh

```diff
@@ -35,30 +48,36 @@ if [ ! -d "vendor" ] || [ ! -f "vendor/autoload.php" ]; then
   chmod +x bin/magento
 fi
 
+mkdir -p pub/media pub/static var
 if [ "$IS_NFS" -ne 0 ]; then
   chown -R "${APP_USER}:${CODE_GROUP}" pub/media pub/static var
-  chmod ug+rw,o-w pub/media pub/static var
+  chmod -R ug+rw,o-w pub/media pub/static var
 else
-  chmod a+rw pub/media pub/static var
+  chmod -R a+rw pub/media pub/static var
```

Comment:
setfacl doesn't work for docker volumes (on OSX, at least). So we can do it on CP where we aren't mounting code into the container but local is a problem.